src/components/Card.js

```diff
@@ -73,19 +78,20 @@ const Wrapper = styled.div`
 
 const Header = styled.header`
   display: grid;
-  grid-auto-columns: 1fr;
-  grid-template-columns: 0.2fr 1.4fr 1.4fr;
+  grid-template-columns: 0.1fr 2fr;
 `;
 
 const Icon = styled.img`
   height: 18px;
   width: auto;
-  margin-top: 14px;
+  margin-top: -5px;
```

Comment:
It's more a question for myself, not a suggestion for improvement 😉Recognized you used margin for the positioning and asked myself if you also could use the `position: absolute` attribute 🤔

Reply:
As I understand, since flexbox and grid were introduced to css "position: absolute" is more likely used for navigation and slideshows 🤔